skopt/optimizer/base.py

```diff
@@ -22,7 +22,7 @@ def base_minimize(func, dimensions, base_estimator,
                   acq_func="EI", acq_optimizer="lbfgs",
                   x0=None, y0=None, random_state=None, verbose=False,
                   callback=None, n_points=10000, n_restarts_optimizer=5,
-                  xi=0.01, kappa=1.96, n_jobs=1):
+                  xi=0.01, kappa=1.96, alpha=1.6, gamma=0.0, n_jobs=1):
```

Comment:
The value of alpha shouldn't be checked to be greater than zero?
Just a quick comment: I think that the default value for alpha should be something like 6. In the paper they define alpha as log(2/delta) and then  perform all the experiments with delta=10^-6 (leading to an alpha of nearly 6).
Then they show that this is quite robust, and plot the regret for various value of delta (from 10^-1 to 10^-9), but I think that if they showed the results only for 10^-6 it should be the one with best results :)

Reply:
![gp_mi_calibrate_a](https://user-images.githubusercontent.com/16848331/31631480-c5d1d664-b287-11e7-9e54-96a89bf33028.png)

Thanks, I missed that they set <a href="https://www.codecogs.com/eqnedit.php?latex=\delta=10^{-6}" target="_blank"><img src="https://latex.codecogs.com/gif.latex?\delta=10^{-6}" title="\delta=10^{-6}" /></a>. Like you said, the regret seems robust to `alpha`, but I wonder why they didn't set <a href="https://www.codecogs.com/eqnedit.php?latex=\delta=10^{-1&space;}" target="_blank"><img src="https://latex.codecogs.com/gif.latex?\delta=10^{-1&space;}" title="\delta=10^{-1 }" /></a>. At least for the experiment shown in the Figure 4 above, it seems to be best. Am I overlooking their justification for that somewhere?